nixos/tests/gatus.nix

```diff
@@ -0,0 +1,33 @@
+import ./make-test-python.nix
+  ({ pkgs, lib, ... }: {
+    name = "gatus";
+    meta.maintainers = with pkgs.lib.maintainers; [ pizzapim ];
+
+    nodes.machine =
+      { ... }:
+      {
+        services.gatus = {
+          enable = true;
+
+          settings = {
+            web.port = 8080;
+            metrics = true;
+
+            endpoints = [{
+              name = "metrics";
+              url = "http://localhost:8080/metrics";
+              interval = "1s";
+              conditions = [
+                "[STATUS] == 200"
+              ];
+            }];
+          };
+        };
+      };
+
+    testScript = /*python*/ ''
```

Comment:
Remove the comment.

Reply:
The comment is to instruct treesitter about the language inside the comment, but I see this is not compatible with formatting :(